### DISCUSSION TITLE
Is GitHub Student Developer Pack only for Tier 1 college students having university email address?

### DISCUSSION THREAD

**Discussion by justraqib** [community/community#55083] [GitHub Education]
### Hi everyone,

I am a student currently pursuing a three year degree course in a Tier 3 college. My college doesn't have the infrastructure to provide email addresses to each of their students. Although GitHub says that they accept dated ID cards, I have been unsuccessful in getting approved despite providing the same. I have been trying for days now, and have spent hours today and uploaded every possible document including all my fees receipts, semester grade cards and dated ID card. GitHub just says sorry on submission and rejects my application without any proper reason or way to fix it. There are multiple community threads with people facing similar issues. Is there any other way for Tier 3 college students to verify themselves without university email addresses?

**Comment by ryanlua** [comment]
Sorry to hear that @justraqib, 

> However, students whose schools don’t provide email addresses are able to apply for the Pack using their dated student ID. 

You do not need a school email to get the GitHub Student Developer Pack as stated in "[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)"

It seems like you are facing common problems with your applications so below I have listed some common problems for you.

- **Check if your school is recognized by GitHub** Check to see if your school is in [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). If not, then you may need to contact GitHub Support for additional support.
- **Use up to date documents/information.** Provide the most up to date information and documents otherwise you might be rejected if it is too outdated.
- **Ensure your GitHub profile matches.** Make sure all information such as your or location both match your profile and documents.
- **Don't use a VPN.** GitHub may detect you using a VPN and automatically deny you access. Try applying without any VPN.
- **Translate your documents.** If your school is in another language than English than you should provide both the original untranslated document and English translated document.

You can read "[Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)" and the "[Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)" discussion for more information about Global Campus rejections. 

I would get in contact with GitHub through "[Get help with GitHub Education](https://support.github.com/contact/education)" and ask for them additional support if this question isn't answered in a while. 

You shouldn't be rejected for not having a school email but again if this amounts to nothing, get in touch with GitHub for help.

**Comment by samus-aran** [comment]
Hi @justraqib 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Yes, we do prefer academic emails but we often accept other forms of evidence if the applied school allows it. Let us know how you get on 👍

**Reply by fatimaamjad01** [reply]
@samus-aran AOA, mam , student status wasn't verified when I applied for the GitHub Student Pack, even though I provided my student email and card. They said they'd send a verification email to my student email, but i didn't receive it. I also gave my fee challan and student ID, but my application got rejected.any solution
